bottlerocket-settings-models/settings-extensions/kubelet-device-plugins/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "settings-extension-kubelet-device-plugins"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
We must also increment the version of `bottlerocket-settings-models/settings-models/Cargo.toml`